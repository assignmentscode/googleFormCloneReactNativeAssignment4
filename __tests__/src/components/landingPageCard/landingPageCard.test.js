import React from 'react';
import renderer from 'react-test-renderer';
import LandingPageCard from '../../../../src/components/landingPageCard/landingPageCard';

describe('MainPage', () => {
  it('snapshot should match the previous snapshot', () => {
    const tree = renderer.create(<LandingPageCard />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
