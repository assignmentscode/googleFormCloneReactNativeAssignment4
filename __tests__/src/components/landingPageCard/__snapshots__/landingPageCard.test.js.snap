// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainPage snapshot should match the previous snapshot 1`] = `
<View
  style={
    Object {
      "backgroundColor": "white",
      "height": 200,
      "marginBottom": 20,
      "marginTop": 20,
    }
  }
>
  <View
    style={
      Object {
        "borderColor": "black",
        "flex": 4,
        "justifyContent": "space-around",
        "paddingLeft": 20,
        "paddingRight": 20,
      }
    }
  >
    <Text
      style={
        Object {
          "alignSelf": "stretch",
          "color": "black",
          "fontSize": 22,
          "fontWeight": "500",
          "textAlign": "left",
        }
      }
    >
      
    </Text>
  </View>
  <View
    style={
      Object {
        "alignItems": "flex-end",
        "borderTopColor": "gray",
        "borderTopWidth": 2,
        "flex": 1,
        "justifyContent": "center",
      }
    }
  >
    <Text
      style={
        Object {
          "alignSelf": "stretch",
          "color": "black",
          "fontSize": 18,
          "fontWeight": "400",
          "paddingRight": 30,
          "textAlign": "right",
        }
      }
    >
      Created On:
       
      
    </Text>
  </View>
</View>
`;
